src/interp.cc

```diff
@@ -2414,6 +2434,13 @@ Result Thread::Run(int num_instructions) {
         break;
       }
 
+      case Opcode::I8X16ExtractLaneS: {
+        v128 dLane = static_cast<v128>(Pop<v128>());
```

Comment:
Better name than `dLane` here? Other examples in this file use `value`.

Reply:
Agree.

Reply:
Done.